docs/api-reference-1/frontity-components.md

```diff
@@ -14,6 +14,81 @@ import Image from "@frontity/components/image";
 
 ## Components
 
+### Link
+
+`<Link />` is a React component that you can use in your Frontity project to define links that works with the internal routing system. * Under the hood, this component uses the `actions.router.set(link)` method from `@frontity/tiny-router` and creates an `<a/>` tag. 
+
+#### Props
+
+| Name | Type | Default | Description | Optional |
+| :--- | :--- | :--- | :--- | :--- |
+| `link` | string | `null` | The URL to link to. | `false` | 
```

Comment:
I don't think saying that the default of the `link` is `null` is correct, because `null` is a valid value in JavaScript and the default of `link` is not `null`. It'd more correct to say that `default` is `undefined`, but I think it would also be confusing. Some props simply have no default value because they are required.

Maybe it could be noted with a hyphen or with `Required`.

| Name   | Type   | Default  | Description         | Optional |
| :----- | :----- | :------- | :------------------ | :------- |
| `link` | string | –––      | The URL to link to. | `false`  |
| `link` | string | Required | The URL to link to. | `false`  |

Reply:
Maybe it'd be better to change the `Optional` for required and move the columns a bit to make this clearer:

| Name     | Type   | Required | Default | Description               |
| :------- | :----- | :------- | :------ | :------------------------ |
| `link`   | string | Yes      | –––     | The URL to link to.       |
| `target` | string | No       | `_self` | The target of the anchor. |

What do you think?

Reply:
I think `undefined` is correct as it's the _default_ value when some variable/property is created but has no specific value assigned. And it's the value you should get when you check this prop when there's no value passed through it

**Default** → `undefined`
**Optional** → `false`

...should be clear enough

Reply:
> And it's the value you should get when you check this prop when there's no value passed through it

Not really because the prop is required so there's no way to check it if you don't pass it. The component will simply not work.

Reply:
Oks. Then I think the approach you suggested in https://github.com/frontity/docs/pull/205#discussion_r476469185 is the best one

Reply:
Okey. Re-request the review once ready! 🙂

Reply:
This is the current table

| Name | Type | Required | Default | Description | 
| :--- | :--- | :--- | :--- | :--- |
| `link` | string | Yes | --- | The URL to link to. |
| `target` | string | No | `_self` | The [target](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#target) of the anchor. Possible values: `_self` or `_blank` | 
| `onClick` | function | No | `undefined` | The `onClick` handler. Can be used to pass an optional callback that will be invoked on click. | 
| `scroll` | boolean | No | `true` | Whether the browser should scroll up to the top upon navigating to a new page.  | 
| `prefetch` | boolean | No | `true` | Whether Frontity should automatically prefetch this link or not. The prefetching mode is controlled through `state.theme.prefetch`  | 
| `aria-current` | string | No | `undefined` | [Indicates the element that represents the current item within a container or set of related elements](https://www.w3.org/TR/wai-aria-1.1/#aria-current) | 


I've changed the default values to `undefined` in those props that doesn't have a defaultValue defined. Is this correct?

Reply:
Oh, yes, that is more correct than `null` because we are not setting those to `null` either 🙂 👍 👍